README.md

```diff
@@ -51,60 +53,47 @@ pip install -e .[all]
 Note: The `-e` option is the so-called "editable" mode. This will allow code changes taking effect immediately.
 
 
-## Quick start / Try out on a simple example
-
-Here, we use a [3D labelfree prediction](https://www.allencell.org/label-free-determination.html#:~:text=The%20Label-Free%20Determination%20model%20can%20leverage%20the%20specificity,structures.%20How%20does%20the%20label-free%20determination%20model%20work%3F) as an example. (If you followed [basic installation](#basic-installation), you will need to install the `quilt3` package by `pip install quilt3`, in order to download the data programmatically.)
+## Quick start
 
-**step 1 (data preparation):**, we pull 100 examples of lamin B1 images and the corresponding brightfield images from the AllenCell quilt bucket by running the following in the command line. 
+You can try out on a simple example following [the quick start guide](tutorials/quick_start.md)
 
-```bash
-# Suppose the current working directory is the root of mmv_im2im
-python  scripts/pull_labelfree_sample_data.py --download_path /path/to/save/the/downloaded/images/ --structure LMNB1 --num 100 
-```
+Basically, you can specify your training configuration in a yaml file and run training with `run_im2im --config /path/to/train_config.yaml`. Then, you can specify the inference configuration in another yaml file and run inference with `run_im2im --config /path/to/inference_config.yaml`. You can also run the inference as a function with the provided API. This will be useful if you want to run the inference within another python script or workflow.  Here is an example:
```

Comment:
@funkyZhaofour read this carefully, you will need this for CD64 segmentation inference.